pkg/cloud/azure/azure_storage_test.go

```diff
@@ -63,6 +63,20 @@ func (a *azureConfig) filePathWithSchemeClientAuth(scheme string, f string) stri
 	return uri.String()
 }
 
+func (a *azureConfig) filePathImplicitAuth(f string) string {
+	return a.filePathWithSchemeClientAuth("azure", f)
+}
+
+func (a *azureConfig) filePathWithSchemeImplicitAuth(scheme string, f string) string {
+	uri := url.URL{Scheme: scheme, Host: a.bucket, Path: f}
+	values := uri.Query()
+	values.Add(AzureAccountNameParam, a.account)
+	values.Add(AzureEnvironmentKeyParam, a.environment)
```

Comment:
can implicit auth also leverage rbac env vars as well? If so, we should test for that too.

Reply:
implicit auth _only_ works with RBAC, not legacy storage keys (it's a cross-azure API, so it doesn't support any product's legacy auth mechanism)

https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet

Reply:
aha, this  got me a little confused in the cloud backup/restore tests. see above.